docs/source/reference/command_line.rst

```diff
@@ -323,11 +323,9 @@ Below is a list with all available subcommands.
                                       .aiida extension for these HTTP addresses. Automatically
                                       discovered archive URLs will be downloaded and added to
                                       ARCHIVES for importing
```

Comment:
Note: I first removed these changes again but then noticed that they also appear on the [CI pre-commit check](https://github.com/ltalirz/aiida-core/runs/1721347516?check_suite_focus=true) - some updated version perhaps?